main.js

```diff
@@ -1 +1,115 @@
-console.log("Tic Tac Toe")
\ No newline at end of file
+// alternatives for var?
+var grid, playerActive, gameGoing, counts;
```

Comment:
Variables declared outside functions have global scope, which means they can be accessed anywhere in your code

Use `const` instead of `var`, you should never use `var`